src/main/java/seedu/address/model/note/Note.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.model.note;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Note in the address book.
+ * Guarantees: immutable; name is valid as declared in {@link #isValidNote(String)}
+ */
+public class Note {
+
+    public static final String MESSAGE_CONSTRAINTS = "Notes should be a String";
+    public static final String VALIDATION_REGEX = "\\p{Print}+"; // printable
```

Comment:
Just wondering if there should be a constraint on the maximum notes string size. I think this will be dependent on the note display behaviour on the PersonCard.

Reply:
Thx, I've added 50 characters constraints to string length. Currently, the note is displayed as simple label text.